### PR TITLE
Added Missing Models

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
This adds a script for checking for missing models, a script for running them and removed older scripts for running models.

It also add a list of models which currently aren't run.

### CODE REVIEW

File: scripts/mmteb/running_model/check_results.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
+"""This script checks that all result have been runs and writes a file: missing_tasks.txt with the missing tasks for each model"""
```
Comment (Muennighoff) [inline scripts/mmteb/running_model/check_results.py:1]:
```suggestion
"""This script checks that all result have been run and writes a file: missing_tasks.txt with the missing tasks for each model"""
```

**Review by Muennighoff — APPROVED** [review]
Really nice! I will look into those missing_tasks 👍

**Comment by Muennighoff** [comment]
Can you update the `scripts/mmteb/running_model/missing_tasks.txt` here based on the latest pushed results?

**Comment by KennethEnevoldsen** [comment]
Updated! Already notably better - running check_results.py should update it as well.

**Comment by Muennighoff** [comment]
Should be mostly there now? Sorry can you run it once more & update the file? 😅

**Comment by KennethEnevoldsen** [comment]
Yep. Will just merge this in an create a new PR

**Comment by KennethEnevoldsen** [comment]
See #1038

**Pull request merged**